src/ucp/proto/proto_single.inl

```diff
@@ -17,10 +17,12 @@ static UCS_F_ALWAYS_INLINE ucs_status_t
 ucp_proto_am_bcopy_single_send(ucp_request_t *req, ucp_am_id_t am_id,
                                ucp_lane_index_t lane,
                                uct_pack_callback_t pack_func, void *pack_arg,
-                               size_t max_packed_size)
+                               size_t max_packed_size, int fast_path)
 {
     ucp_ep_t *ep                       = req->send.ep;
     const uct_iface_attr_t *iface_attr = ucp_ep_get_iface_attr(ep, lane);
+    uct_ep_h uct_ep = fast_path ? ucp_ep_get_fast_lane(ep, lane) : 
```

Comment:
do not init inline to valign =

Reply:
not sure I understand: the issue is alignment or inline initialization?

Reply:
init non-inline so need to valign =